packages/react-components/src/organisms/MainNavigation.tsx

```diff
@@ -69,9 +78,16 @@ const MainNavigation: React.FC<MainNavigationProps> = ({
       </li>
       <li>
         <NavigationLink href={newsAndEventsHref} icon={newsIcon}>
-          News and Events
+          {isEnabled('EVENTS_PAGE') ? 'News' : 'News and Events'}
```

Comment:
This needs test (regression and non-regression that distinguish) - or you could ask whether changing to `News` immediately is okay